lib/seahorn/BvOpSem2MemRepr.cc

```diff
@@ -52,6 +52,9 @@ Expr OpSemMemArrayRepr::MemCpy(Expr dPtr, Expr sPtr, unsigned len,
       Expr val = op::array::select(srcMem, sIdx);
       res = op::array::store(res, dIdx, val);
     }
+  } else {
+    ERR << "Word size and pointer are not aligned!";
+    llvm_unreachable(nullptr);
```

Comment:
We had a habit of using `llvm_unreachable` for `assert(false)`. It ended up a bad idea because of how it is optimized. Let's use `assert(false);` instead

Reply:
Ack. Added assert